modules/migration-known-issues.adoc

```diff
@@ -26,12 +26,16 @@ These annotations preserve the UID range, ensuring that the containers retain th
 
 * If you select the data verification option for PVs that are migrated with the filesystem copy method, performance is significantly slower. Velero generates a checksum for each file and checks it when the file is restored.
 
+ifeval::["{mtc-version}" == "1.2.4"]
+
 ifdef::migrating-3-4[]
-* In the current release (CAM 1.2), you cannot migrate from {product-title} 3.7 to {product-version} because certain API `GroupVersionKinds` (GVKs) that are used by the source cluster are deprecated. You can manually update the GVKs after migration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1817251[*BZ#1817251*])
+* You cannot migrate from {product-title} 3.7 to {product-version} because certain API `GroupVersionKinds` (GVKs) that are used by the source cluster are deprecated. You must manually update the GVKs after migration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1817251[*BZ#1817251*])
```

Comment:
This one's a bit confusing. Maybe s/You cannot migrate from {product-title} 3.7 to {product-version}/Migrating from {product-title} 3.7 to {product-version} requires extra steps

Reply:
Agree. I'll make it clearer.